services/statestorage/test/_stress_test.go

```diff
@@ -0,0 +1,79 @@
+package test
+
+import (
+	"context"
+	"fmt"
+	"github.com/orbs-network/orbs-network-go/test"
+	"github.com/orbs-network/orbs-network-go/test/builders"
+	"github.com/orbs-network/orbs-spec/types/go/services"
+	"github.com/stretchr/testify/require"
+	"math/rand"
+	"runtime"
+	"testing"
+	"time"
+)
+
+const TpxAvg = 100
```

Comment:
I think our convention for consts (which is different than go's default) is: `TPX_AVG`

Reply:
conformed